source/includes/_fonenumbers.md

```diff
@@ -18,7 +18,7 @@ $ curl --request GET
 ```json
 {
   "statusCode": 200,
-  "dids": [
+  "Fonenumbers": [
```

Comment:
can we merge this? I will fix it in a different branch. All these fonenumber responses have changed.